src/styles/global.scss

```diff
@@ -1,34 +1,75 @@
-@import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,300;0,400;0,500;0,600;0,700;1,300;1,400;1,500;1,600;1,700&display=swap");
-
 @import "~@angular/flex-layout/mq";
 @import "reset.scss";
 @import "colors.scss";
 @import "editor.scss";
-html,
-body {
-  height: 100%;
-}
+
 body {
-  margin: 0;
-  font-family: "Roboto", "Helvetica Neue", sans-serif;
   background: #eef5f9;
-  // overflow: hidden;
 }
 
-h1 {
-  background: #fff;
-  box-shadow: 0 2px 6px 0 #6494ff26;
-  padding-left: 8px;
-  color: $primary;
+/* XS (extra small)*/
+.container {
+  margin: 0 16px;
+  width: auto;
+}
+
+/* SM (small) */
+@media (min-width: 600px) {
+  .container {
+    margin: 0 32px;
+    width: auto;
+  }
+}
+
+/* MD (medium) */
+@media (min-width: 905px) {
+  .container {
+    margin: 0 auto;
+    width: 840px;
+  }
+}
 
-  @include layout-bp(lt-md) {
-    font-size: 18px;
+/* LG (large) */
+@media (min-width: 1240px) {
+  .container {
+    margin: 0 200px;
+    width: auto;
   }
-  > div {
-    min-height: 70px;
+}
+
+/* XL (extra large) */
+@media (min-width: 1440px) {
+  .container {
+    margin: 0 auto;
+    width: 1040px;
   }
 }
 
+.mt-16 {
+  margin-top: 16px;
+}
+
+.ptb-8 {
```

Comment:
I suggest we use `py` instead of `ptb` and `px` instead of `prl` (or would it be `plr`?) because bootstrap and bulma use that too

Reply:
I agree. Made the change.